test/voice-broadcast/utils/shouldDisplayAsVoiceBroadcastRecordingTile-test.ts

```diff
@@ -47,23 +51,32 @@ describe("shouldDisplayAsVoiceBroadcastRecordingTile", () => {
         client = createTestClient();
     });
 
-    describe.each(testCases)(
-        "when called with user »%s«, sender »%s«, state »%s«",
-        (userId: string, senderId: string, state: VoiceBroadcastInfoState, expected: boolean) => {
+    describe.each<TestTuple>(testCases)(
+        "when called with user »%s«, sender »%s«, device »%s«, sender device »%s« state »%s«",
+        (userId, senderId, deviceId, senderDeviceId, state, expected) => {
             beforeEach(() => {
-                event = mkEvent({
-                    event: true,
-                    type: VoiceBroadcastInfoEventType,
-                    room: "!room:example.com",
-                    user: senderId,
-                    content: {},
-                });
+                event = mkVoiceBroadcastInfoStateEvent("!room:example.com", state, senderId, senderDeviceId);
                 mocked(client.getUserId).mockReturnValue(userId);
+                mocked(client.getDeviceId).mockReturnValue(deviceId);
             });
 
             it(`should return ${expected}`, () => {
                 expect(shouldDisplayAsVoiceBroadcastRecordingTile(state, client, event)).toBe(expected);
             });
         },
     );
+
+    it("should return false, when all params are null", () => {
+        // @ts-ignore
+        event = mkVoiceBroadcastInfoStateEvent("!room:example.com", null, null, null);
+        // @ts-ignore
+        expect(shouldDisplayAsVoiceBroadcastRecordingTile(null, client, event)).toBe(false);
+    });
+
+    it("should return false, when all params are undefined", () => {
+        // @ts-ignore
+        event = mkVoiceBroadcastInfoStateEvent("!room:example.com", null, null, null);
+        // @ts-ignore
+        expect(shouldDisplayAsVoiceBroadcastRecordingTile(null, client, event)).toBe(false);
```

Comment:
should the types of this function be updated to accept a `null` state? maybe you could add a comment explaining what these tests cover/why the ts-ignores

Reply:
Could make sense. It is a test function where you eventually want to create broken data.